setup.py

```diff
@@ -192,13 +192,13 @@ def get_long_description():
         "bottleneck": ["LICENSE"],
         "bottleneck.tests": ["data/*/*"],
     },
-    requires=["numpy"],
-    install_requires=["numpy"],
+    install_requires=["numpy>=1.25"],
```

Comment:
why so high ? numpy `1.21.2` is the oldest version with Python 3.10 support and building against numpy 2.0.0rc1 gives you backward compatible down to 1.19 for free

Reply:
Ah ok for some reason I thought it was higher. I'll remove this change since I don't know what the minimum supported version actually is for bottleneck.